omnibus/config/software/elasticsearch.rb

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright 2019 Chef Software, Inc.
+# Copyright:: Chef Software, Inc.
```

Comment:
should this be 2021?

Reply:
We actually don't need dates here since the copyright is some massive length now. We've removed them in most places and I nuke them as I see them. The :: is an rdoc thing in chef that Adam sprinkled everywhere.